### DISCUSSION TITLE
Authentication issues on HOOBS

### DISCUSSION THREAD

**Discussion by mplopez99** [dgreif/ring#1196] [Q&A]
Running HOOBS on a Pi 4B -install the Ring plugin as its own hub. Log in and verify with 2FA but still have the authentication error. response code 406. Not sure why it says plug in is not configured.

![image](https://user-images.githubusercontent.com/123312479/230398856-8db249e8-e8a2-4ed0-9e52-41e246c374c6.jpg)

HOOBS VER= 4.3.0
Ring Ver = 11.7.4

Any help would be greatly appreciated!

**Comment by dgreif** [comment]
This should be fixed in 11.7.5
